app/livechat/client/views/app/tabbar/visitorInfo.js

```diff
@@ -299,11 +299,13 @@ Template.visitorInfo.onCreated(function() {
 
 	if (rid) {
 		this.autorun(() => {
-			const room = Rooms.findOne({ _id: rid });
-			this.room.set(room);
-			this.visitorId.set(room && room.v && room.v._id);
-			this.departmentId.set(room && room.departmentId);
-			this.tags.set(room && room.tags);
+			const sub = this.subscribe('livechat:rooms', { _id: rid });
```

Comment:
the `subscribe` can be moved to outside the `autorun`, so only the find needs to be inside the `autorun` to update the reactive vars whenever the room data changes.